diff --git a/src/cryptonote_basic/hardfork.cpp b/src/cryptonote_basic/hardfork.cpp
index e773bcd5..38648026 100644
--- a/src/cryptonote_basic/hardfork.cpp
+++ b/src/cryptonote_basic/hardfork.cpp
@@ -132,9 +132,8 @@ HardFork::ParamsIterator HardFork::get_hardcoded_hard_forks(network_type nettype
   return {nullptr, nullptr};
 }
 
-HardFork::HardFork(cryptonote::BlockchainDB &db, uint8_t original_version, std::chrono::seconds forked_time, uint64_t window_size, uint8_t default_threshold_percent):
+HardFork::HardFork(cryptonote::BlockchainDB &db, uint8_t original_version, uint64_t window_size, uint8_t default_threshold_percent):
   db(db),
-  forked_time(forked_time),
   window_size(window_size),
   default_threshold_percent(default_threshold_percent),
   original_version(original_version),
diff --git a/src/cryptonote_basic/hardfork.h b/src/cryptonote_basic/hardfork.h
index 2b971d42..9a1205ee 100644
--- a/src/cryptonote_basic/hardfork.h
+++ b/src/cryptonote_basic/hardfork.h
@@ -56,7 +56,6 @@ namespace cryptonote
       Ready
     } State;
 
-    constexpr const static auto DEFAULT_FORKED_TIME = 31557600s; // a year in seconds
     static const uint64_t DEFAULT_WINDOW_SIZE = 10080; // supermajority window check length - a week
     static const uint8_t DEFAULT_THRESHOLD_PERCENT = 80;
 
@@ -74,12 +73,10 @@ namespace cryptonote
      * @brief creates a new HardFork object
      *
      * @param original_version the block version for blocks 0 through to the first fork
-     * @param forked_time the time in seconds before thinking we're forked
-     * @param update_time the time in seconds before thinking we need to update
      * @param window_size the size of the window in blocks to consider for version voting
      * @param default_threshold_percent the size of the majority in percents
      */
-    HardFork(cryptonote::BlockchainDB &db, uint8_t original_version = 1, std::chrono::seconds forked_time = DEFAULT_FORKED_TIME, uint64_t window_size = DEFAULT_WINDOW_SIZE, uint8_t default_threshold_percent = DEFAULT_THRESHOLD_PERCENT);
+    HardFork(cryptonote::BlockchainDB &db, uint8_t original_version = 1, uint64_t window_size = DEFAULT_WINDOW_SIZE, uint8_t default_threshold_percent = DEFAULT_THRESHOLD_PERCENT);
 
     /**
      * @brief add a new hardfork height
@@ -266,7 +263,6 @@ namespace cryptonote
 
     BlockchainDB &db;
 
-    std::chrono::seconds forked_time;
     uint64_t window_size;
     uint8_t default_threshold_percent;
 
